roles/openshift_control_plane/tasks/encrypting_config.yml

```diff
@@ -0,0 +1,54 @@
+---
+- when: openshift_encryption_config is defined
```

Comment:
put this when condition on the import_tasks instead of making this whole file a block.